redis_json/Cargo.toml

```diff
@@ -25,9 +25,11 @@ ijson.workspace = true
 serde_json.workspace = true
 serde.workspace = true
 libc = "0.2"
-redis-module = "2.0"
+redis-module ={ version = "^2.0.7", default-features = false, features = ["min-redis-compatibility-version-7-2"] }
```

Comment:
No need to work on version less than 7.2?

Reply:
I think not.
We need to update ramp to "7.2"
> min_redis_version: "7.1"

Maybe also add a min version check on module load for OSS?

@adrianoamaral

Reply:
we should consider attainment with the min grouped feature set. So, should follow 7, in this case, and be back-ported also for the preview version, but it won't be JSON 2.6, but 2.4